python/eos/data/native.py

```diff
@@ -274,7 +274,9 @@ def create(path, parameters, samples, weights, proposal, sigma_test_stat=None):
         :param samples: Samples as a 2D array of shape (N, P).
         :type samples: 2D numpy array
         :param weights: Weights on a linear scale as a 2D array of shape (N, 1).
-        :type weights: 1D numpy array, optional
+        :type weights: 1D numpy array
```

Comment:
Hmm this was not supposed to end up in this commit. I will move it to the previous commit together with your other comments if any.